ProgLabHW4/Event.py

```diff
@@ -0,0 +1,23 @@
+
+class Event:
+
+    amount = 1
+
+    def __init__(self, g_title, g_rest, g_date_and_time, g_duration, g_price):
+        self.id = Event.amount
+        self.title = g_title
+        self.rest_name = g_rest
+        self.date_time = g_date_and_time
+        self.duration = g_duration
+        self.price = str(g_price) + ' $'
+        Event.amount += 1
+
+    def __str__(self):
+        to_return = 'ID: ' + str(self.id) + '\n'

```

Comment:
use cycle here, method for getting property by name and dictionary (model of event)